fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/GenericResult.java

```diff
@@ -23,6 +24,16 @@
      */
     protected List<T> errors;
 
+    public GenericResult() {
+        isSuccess = true;
+        errors = new ArrayList<T>();
```

Comment:
In that case, I think a special immutable empty list would be nice if there is no error. I notice your team really spends a lot of effort on refining the library, that is awesome! But due to time difference, we cannot talk in person at the same time. But as soon as I fully get your points, I will accept PR or try to release a SNAPSHOT version for you guys to check.